_posts/2023-11-16-virginia-beach-city/virginia-beach-city.Rmd

```diff
@@ -0,0 +1,229 @@
+
+| Demographic              | Population | Voting Age Pop. | Citizen Voting Age Pop. |
+|-------------------|------------------|------------------|------------------|
+| White, non-Hispanic      | 58.7%      | 61.5%           | 65.8%                   |
+| Black, non-Hispanic      | 20.1%      | 20.1%           | 19.7%                   |
+| Asian, non-Hispanic      | 10%        | 9.4%            | 6.8%                    |
+| Hispanic                 | 8.8%       | 7.6%            | 6.4%                    |
+| Black + Hispanic + Asian | 40.5%      | 37.2%           | 32.8%                   |
+
+: Table 2: Demographic Breakdown of Virginia Beach, VA. in 2020
+
+Figure 6 shows the population density of the overall city to a precinct level. This is important for drawing city council boundaries, as per the Virginia Voting Rights Act, a district can only deviate from the average population in every district by 5%. However, how do we know who is actually eligible to vote? Figure 7 shows the percentage of the population in any given precinct that is of voting age, which is more indicative of current voter demographics.
+
+::: {style="display: flex;"}
+::: {style="flex-direction: column; margin-right: 10px;"}
+![Figure 6: *Population Density by Precinct*](popmap.svg)
+:::
+
+::: {style="flex-direction: column;"}
+![Figure 7: *Percent Voting Age Population by Precinct*](vappop.svg)
+:::
+:::
+
+Figures 8 and 9 show the percentage of minority and black populations by precinct. Districts 4, 7, and 10 are majority-minority districts (MMDs), shown in Figure 2.
+
+::: {style="display: flex;"}
+::: {style="flex-direction: column; margin-right: 20px;"}
+![Figure 8: *Minority Population Percentage by Precinct*](fig3mppprecinct.svg){width="328"}
+:::
+
+::: {style="flex-direction: column;"}
+![Figure 9: *Black Population Percentage by Precinct*](fig4blackpct.svg){width="328"}
+:::
+:::
+
+In Virginia Beach City, minority populations tend to vote as a block for Democratic candidates. Figure 10 displays the vote percentage of Democrats and Republicans in each precinct; Democrats perform best in black/minority precincts. To quantify how cohesive a group of voters are, Figure 11 examines only precincts where the majority of residents in the precinct is a minority. The percentage of democratic votes in any given precinct trends similarly to the overall percentage of minority residents in the precinct, ranging from a .78 to 1.05 ratio.
+
+::: {style="display: flex;"}
+::: {style="flex-direction: column; margin-right: 20px;"}
+![Figure 10: *Minority Population Percentage by Precinct*](fig5demvote.svg){width="328"}
+:::
+
+::: {style="flex-direction: column;"}
+![Figure 11: *Percent of Democratic Votes Compared to Minority Population Percentages in Majority Minority Precincts*](fig6demperf.svg){width="328"}
```

Comment:
Very interesting figure, although I'm curious how you would interpret these results.